src/schema/nmdc.yaml

```diff
@@ -2939,6 +2945,17 @@ slots:
     mixins: object_set
     range: LibraryPreparation
     description: This property links a database object to the set of DNA extractions within it.
+  metatranscriptome_assembly_set:
```

Comment:
Is it possible to be more consistent with set_name and type here:
- using `Activity` in the type name(s) in a consistent way
- consistency between type and set_name in a CamelCase -> snake_case manner

Reply:
Im not using Activity for new classes to minimize changes needed for a backmerge with berkeley.